2022/Handel signature aggregation in hotStuff.md

```diff
@@ -0,0 +1,40 @@
+# Implementing Different signature aggregations in HotStuff
+
+## Administrative 
+
+- Supervisor: Arian Baloochestani
+- Responsible Professor: Leander Jehl
+
+## Prerequisites
+
+ - DAT650 Blockchains
```

Comment:
There will probably not be students that already have taken the subject. 
You can write that they should have taken DAT530Distributed systems and that they should be good go programmers.
You can write that it is recommended they do take the blockchain course concurrently.